docs/services/postgrest.md

```
@@ -0,0 +1,241 @@
+    )
+
+    curl -X POST http://host-1:3100/products \
+        -H "Content-Type: application/json" \
+        -H "Authorization: Bearer $TOKEN" \
+        --data '{"name": "Widget", "price": 9.99}'
+    ```
+
+The `role` claim must name a PostgreSQL role granted to the PostgREST
+authenticator. Grant the role before sending authenticated requests:
+
+```sql
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)